### PR TITLE
update de.po

### PR DESCRIPTION
**Pull request by bilddateien**
Some corrections (module "filmic" is named "filmisch" in german version, so corrected all tooltips to be inline with that) and some minor improvements

### CODE REVIEW

**Comment by MStraeten** [comment]
I think "Filmisch" is no proper translation for filmic. It's not even valid in this context since "Filmisch" means "like a movie". So there's a demand to find a better translation. Maybe a term describing what's happening there ...

**Comment by bilddateien** [comment]
I would agree with that 100%. But as a small casual translator, I took this higher-level naming as a given and limited myself to making the rest of the translation consistent.
To be honest: at the introduction of the module I had briefly wondered about the name, but quickly got used to it :)
The problem seems to me to be finding an apt paraphrase in _one_ term (possibly also compound noun) - and I haven't really come up with anything so far that I could suggest.

**Comment by johnny-bit** [comment]
Filmic is a name based on what it does - "Film Emulsion-Like Camera Rendering Transforms". Calling it felcrt would be silly ;) and indeed in english "filmic" means "film-like", this meaning "like analog camera film". Maybe this helps with translation

**Comment by bilddateien** [comment]
So from what you say I draw the conclusion:
- filmic (english) refers to _film-like_
- If we translate _filmic_ to _filmisch_ - this would result in something like _cinematic_ if we would retranslate this back to english ... 
- so _filmic_ in German would be something like _Film-artig_

**Comment by MStraeten** [comment]
Maybe better to have a noun to describe what the module does or can be used for (like colorcalibration for the channel mixer)
Maybe „Filmartige Dynamikkompression“  or just „Dynamikkompression“ but thats just a subset of waht it does.
Unfortunately there’s no translation for it in Blender documentation - even in german documents the term  Filmic is used. 
so i opt for no translation for „Filmic“ since there won’t be a substantive describing every aspect of the module (or even a generalization of what it does)

**Comment by TurboGit** [comment]
As there is some discussion, please ping me when you've settle on a final translation.

**Comment by bilddateien** [comment]
OK, now we have two options:
- we use the original term _filmic_ in the German version without further translation. That would mean:
    - I undo my changes in this respect and 
    - also _change the module name filmic_ (capital letter at the beginning of the word?)
- we find a better name for the module. Following your suggestions, after some consideration, I came up with the term "Filmkurve" (yes, it's a synthetical word) for two reasons:
    - the module replaces the old module _base curve_.
    - the module simulates the transfer curve of analog film

    and it 
    - also fits to the other names like Farbkurve und RGB-Kurve
    - makes the name break to the current name not too big for the users (compared to "Dynamikkompression" for example)

Opinions on this?




Translated with www.DeepL.com/Translator (free version)

**Comment by MStraeten** [comment]
i opt for using "Filmic RGB" for the module name (even with capital F for a consistent UI) like a trademark for it ;) 
For Beginners its even more consistent to mostly english tutorials for the scene referred workflow ...

"Filmkurve" doesn't seem to be a good solution since Filmic is quite more than just a replacement for basecurve or rgb/lab tonecurve. If there isn't a common german term in graphic or photographic context it doesn’t make sense to create a new synthetical one that needs to be explained ...

**Comment by bilddateien** [comment]
I would like to know how we can move forward here?
- if we stay with the suggestion of @MStraeten: do we need to coordinate this with the translators of the documentation (how to organize this)?
- if the proposal is accepted - do I have to submit a new PR? I guess this one can't be changed and must be rejected then?

**Comment by MStraeten** [comment]
@rabauke since you had some corrections for my translations - do you have an opinion on this?

File: po/de.po [hunk 1]
```diff
@@ -13746,12 +13744,12 @@ msgstr ""
...
 #: ../src/iop/retouch.c:2235
 msgid "type for the blur algorithm"
-msgstr "Typ des Weichzeichnungsalgorithmus"
+msgstr "Typ des Weichzeichner-Filters"
 
 # Ugh ...
 #: ../src/iop/retouch.c:2239
 msgid "radius of the selected blur type"
-msgstr "Radius des selektierten Weichzeichnungstyp"
+msgstr "Radius des ausgewählten Weichzeichner-Filter"
```
Comment (da-phil) [inline po/de.po:13752]:
```suggestion
msgstr "Radius des ausgewählten Weichzeichner-Filters"
```

File: po/de.po [hunk 2]
```diff
@@ -1380,12 +1380,12 @@ msgstr "Belichtung"
...
 #: ../build/lib/darktable/plugins/introspection_retouch.c:379
 msgid "blur type"
-msgstr "Weichzeichnungstyp"
+msgstr "Weichzeichner-Filter"
 
 #: ../build/lib/darktable/plugins/introspection_retouch.c:244
 #: ../build/lib/darktable/plugins/introspection_retouch.c:383
 msgid "blur radius"
-msgstr "Weichzeichnungsradius"
+msgstr "Weichzeichnersradius"
```
Comment (da-phil) [inline po/de.po:1388]:
```suggestion
msgstr "Weichzeichnungsfilter Radius"
```

File: po/de.po [hunk 3]
```diff
@@ -1380,12 +1380,12 @@ msgstr "Belichtung"
 #: ../build/lib/darktable/plugins/introspection_retouch.c:238
 #: ../build/lib/darktable/plugins/introspection_retouch.c:379
 msgid "blur type"
-msgstr "Weichzeichnungstyp"
+msgstr "Weichzeichner-Filter"
```
Comment (da-phil) [inline po/de.po:1383]:
```suggestion
msgstr "Weichzeichnungsfilter"
```

File: po/de.po [hunk 4]
```diff
@@ -13746,12 +13744,12 @@ msgstr ""
 
 #: ../src/iop/retouch.c:2235
 msgid "type for the blur algorithm"
-msgstr "Typ des Weichzeichnungsalgorithmus"
+msgstr "Typ des Weichzeichner-Filters"
```
Comment (da-phil) [inline po/de.po:13747]:
```suggestion
msgstr "Typ des Weichzeichnungsfilters"
```
This seems to be a proper term, it is used in academic literature and also in photoshop :stuck_out_tongue:

**Review by da-phil — COMMENTED** [review]
Just some minor correction proposals, I'm also not a big fan of those "Immer nachfragen, ..." translations with the comma, IMHO the comma can go away, but I guess it's a matter of personal preference ;)

File: po/de.po (hunk 2 above)
Comment (rabauke) [reply po/de.po:1388]: I think "Weichzeichnungsfilter Radius" and "Weichzeichner Radius" are not proper German translations of the English open (spaces) compound noun "blur radius". A compound noun should be used for the German translation too. "Weichzeichner Radius" are just two German words. 

The proposed term "Weichzeichnersradius" contains a superficial s, should be "Weichzeichnerradius".  One might argue that the long German compound nouns are hard to read in the UI. Therefore, "Radius des Weichzeichners" might be considered as an alternative (in combination with "Typ des Weichzeichners" for "type for the blur algorithm", see suggestion by @da-phil below.).

Comment (bilddateien) [reply po/de.po:1388]: Thank you.
Also, do you have any other opinions on the "filmisch" vs. "filmic" vs. (maybe, or maybe not) "Filmkurve" discussion at the beginning of this discussion?

Comment (rabauke) [reply po/de.po:1388]: "Filmkurve" sounds a little bit artificial to me. Through, it still much better as all alternatives that I can think of. I particularly dislike the verbatim translation "filmisch" of the English term "filmic". Both terms commonly refer to characteristics of the *artistic genres* film and photography, see 
* https://dictionary.cambridge.org/de/worterbuch/englisch/filmic
* https://www.collinsdictionary.com/dictionary/english/filmic
* https://www.duden.de/rechtschreibung/filmisch

I can hardly imagine that a German native speaker associates "filmisch" with *technical* aspects of analog film. Therefore, the name "filmisch" is not very useful for explaining what the filmic module is about (emulating the response curve of analog film).

I thought about names as "Filmemulation" ("film emulation") or "Tonkurve" (tone curve). But these terms have their own drawbacks. "Filmemulation" usually refers to the emulation of *specific* analog films, for "Tonkurve" I would expect a general curve tool as we have it in dt as well with the Lab and RGB curves.

Thus, I would vote for "Filmkurve".

**Comment by johnny-bit** [comment]
If you rebased and could fix issue #7979 with your PR it would be great :)

File: po/de.po (hunk 2 above)
Comment (da-phil) [reply po/de.po:1388]: I'd also go with the proposal of @rabauke "Weichzeichnerradius" for this correction.

**Comment by TurboGit** [comment]
Ok closing, there has been some more changes since then in de.po.

**Pull request closed without merging**